docs/movement.md

```diff
@@ -0,0 +1,35 @@
+# How to move an artifact signed by tuf-notary
+
+There are a few scenarios for moving artifacts covered in this document:
+* Mirroring the artifact and TUF metadata
+* Copying the image without the original signature (or from a registry that doesn't use TUF)
+* Copying the image and the original signature
+* Copying the image, original signature, and adding an additional verifier's signature
```

Comment:
One more to copy a subset of the signatures. 
For instance, you might have the original signature and a dev verification signature, with a staging signature. When moved into production the dev and maybe staging signatures are dropped as they aren't needed in production.
Think of the signatures as signed claims, and you don't need, or necessarily want to promote all of them